test/jttt/Core/Players/PlayerFactoryTest.java

```diff
@@ -22,14 +23,15 @@ public void setUp() throws Exception {
 
     @Test
     public void invalidGameTypeChoice() {
-        Assert.assertEquals(false, playerFactory.validPlayerTypes(0));
+        Assert.assertEquals(false, playerFactory.isValidGameType(0));
     }
 
     @Test
     public void validGameTypeChoice() {
-        Assert.assertEquals(true, playerFactory.validPlayerTypes(1));
-        Assert.assertEquals(true, playerFactory.validPlayerTypes(2));
-        Assert.assertEquals(true, playerFactory.validPlayerTypes(3));
+        Assert.assertEquals(true, playerFactory.isValidGameType(1));
```

Comment:
Similar to above, these seem like magic numbers. Can we give them a name, or perhaps use an enum?